sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -767,6 +771,7 @@ object FunctionRegistry {
     expression[EqualNull]("equal_null"),
     expression[HllSketchEstimate]("hll_sketch_estimate"),
     expression[HllUnion]("hll_union"),
+    expression[UnwrapUDT]("unwrap_udt"),
```

Comment:
`unwrap_udt` is also ML-specific

Reply:
It is a public method.

Reply:
I see, but `UnwrapUDT` was introduced for [3rd ML library xgboost](https://github.com/apache/spark/pull/36408#issue-1220627011) 

I am not sure whether it is suitable to be exposed in SQL. also cc @WeichenXu123

Reply:
Moved this out.